lib/java/persistance/src/main/java/org/enso/persist/PerMap.java

```diff
@@ -10,7 +10,7 @@
 
 final class PerMap {
 
-  private static final int serialVersionUID = 8652; // Use PR number
+  private static final int serialVersionUID = 11399; // Use PR number
```

Comment:
The PR #11399 did not bump the serial version UID.

I thought that the way `versionStamp` is computed means that it does not need to, but perhaps it was still needed?
Or perhaps it was due to changes in `BindingsMap` that is now serialized without the `typ` reference?

Reply:
> I don't think this change is necessary.

@JaroslavTulach How can the cache system report failures when running `ImportStandardLibrariesBenchmark` when it was preceded by `buildEngineDistribution` that [compiled all the stdlibs](https://github.com/enso-org/enso/actions/runs/12680614808/job/35342759884?pr=12023#step:7:2003)?

Reply:
- I guess #12027 should provide the answer...